tests/dummy/app/components/sample-toolbar-responsive.hbs

```diff
@@ -46,7 +46,14 @@
   <:side as |Tb|>
```

Comment:
Hmm, not necessarily something for now, but this side menu is getting crowded and isn't handling fitting content well... 
![image](https://github.com/lblod/ember-rdfa-editor/assets/1323250/7d9cfe62-4b18-4962-984e-ca88a5d433e9)

Reply:
Agreed, we should probably tackle this as part of this ticket: https://binnenland.atlassian.net/browse/GN-4737?atlOrigin=eyJpIjoiNGFlNmNkMzRlY2JmNDcxNWE4NGQyMGNmMTdlNDFhMjgiLCJwIjoiaiJ9 (redesign of the toolbar)